velox/expression/Expr.h

```diff
@@ -586,7 +586,7 @@ class ExprSetSimplified : public ExprSet {
   ExprSetSimplified(
       const std::vector<core::TypedExprPtr>& source,
       core::ExecCtx* FOLLY_NONNULL execCtx)
-      : ExprSet(source, execCtx, /*enableConstantFolding*/ false) {}
+      : ExprSet(source, execCtx, /*enableConstantFolding*/ true) {}
```

Comment:
This change seems from enabling constant folding in simplified path. Let's remove.